### ISSUE TITLE
CSS modules prevents outside days click

### ISSUE THREAD

**Issue by SammyM** [gpbl/react-day-picker#449]
When using CSS modules and `enableOutsideDays={true}`, I would expect that when clicking on an outside day, it would take me to the next month. This however doesn't seem so be doing anything.

I'm not sure if this is related, but when logging the `modifiers` in the click function, it logs this without CSS modules:
```
Object {outside: true}
```
and this with CSS modules:
```
Object {_26d7gZ-calendar-outside: true}
```

I'm not sure how to create this in a codepen, so sorry for the lack of example.

**Comment by SammyM** [comment]
Thank you! I appreciate the incredibly quick turnaround on this.

**Comment by gpbl** [comment]
😊 published as `v6.1.0`
